.eslintrc

```diff
@@ -207,53 +222,71 @@
     "no-inline-comments": 1,        // disallow comments inline after code (off by default)
     "no-lonely-if": 1,              // disallow if as the only statement in an else block (off by default)
     "no-mixed-spaces-and-tabs": 2,  // disallow mixed spaces and tabs for indentation
     "no-multiple-empty-lines": 0,   // disallow multiple empty lines (off by default)
-    "no-nested-ternary": 1,         // disallow nested ternary expressions (off by default)
+    "no-nested-ternary": 2,         // disallow nested ternary expressions (off by default)
     "no-new-object": 0,             // disallow use of the Object constructor
-    "no-space-before-semi": 0,      // disallow space before semicolon
+    "semi-spacing": 2,              // disallow space before semicolon
     "no-spaced-func": 0,            // disallow space between function identifier and application
     "no-ternary": 0,                // disallow the use of ternary operators (off by default)
-    "no-trailing-spaces": 0,        // disallow trailing whitespace at the end of lines
+    "no-trailing-spaces": 2,        // disallow trailing whitespace at the end of lines
     "no-underscore-dangle": 0,      // disallow dangling underscores in identifiers
-    "no-wrap-func": 0,              // disallow wrapping of non-IIFE statements in parens
-    "one-var": 0,                   // allow just one var statement per function (off by default)
+    "object-curly-spacing": [      // enforce consistent spacing inside braces
+      2,
+      "always"
+    ],
+    "one-var": [                    // allow just one var statement per function (off by default)
+      2,
+      "never"
+    ],
     "operator-assignment": 0,       // require assignment operator shorthand where possible or prohibit it entirely (off by default)
     "padded-blocks": 0,             // enforce padding within blocks (off by default)
-    "quote-props": 0,               // require quotes around object literal property names (off by default)
-    "quotes": 0,                    // specify whether double or single quotes should be used
-    "semi": 0,                      // require or disallow use of semicolons instead of ASI
+    "quote-props": [                // require quotes around object literal property names (off by default)
+      2,
+      "consistent-as-needed",
+      { "keywords": true }
+    ],
+    "quotes": 2,                    // specify whether double or single quotes should be used
+    "semi": 2,                      // require or disallow use of semicolons instead of ASI
     "sort-vars": 0,                 // sort variables within the same declaration block (off by default)
-    "space-after-function-name": 0, // require a space after function names (off by default)
-    "space-after-keywords": 0,      // require a space after certain keywords (off by default)
+    "space-before-function-paren": 2,// require a space after function names (off by default)
```

Comment:
No thanks. This is a super opinionated rule that I do not care to enforce.

Reply:
Eslint is supposed to be opinionated to enforce consistency, I can switch it to forbid the space if you want that

Reply:
Sure, it's more that I don't care for it adding a space. We can enforce a no-space/inverse of it though.